### PR TITLE
Added custom navbar to the project Example

### PR DESCRIPTION
**Pull request by pgonzalez-santiago**
I have added some customs navbar in the example project just to test some stuff.

### CODE REVIEW

**Comment by aksonov** [comment]
Sorry, previous PR make conflicts, could you resolve it? Does Example work well with custom navbars? (If no, I should not merge but we need to fix bugs first)

**Comment by mcabs3** [comment]
@pgonzalez-santiago is this something you are going to update? Otherwise I can close this PR.

**Comment by pgonzalez-santiago** [comment]
Done. Sorry for the delay.

**Comment by pgonzalez-santiago** [comment]
Hey @aksonov sorry I didn't answer you to "Does Example work well with custom navbars?" Custom navbar works perfectly.

**Comment by mcabs3** [comment]
@pgonzalez-santiago can you pull master and make sure everything is good to go? I will then review it and get this in for you.

**Comment by pgonzalez-santiago** [comment]
@mcabs3 Done.

File: Example/components/TabView.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ class TabView extends React.Component {
         <Button onPress={() => { Actions.tab_4(); }}>Switch to tab4</Button>
         <Button onPress={() => { Actions.tab_5({ data: 'test!' }); }}>Switch to tab5 with data</Button>
         <Button onPress={() => { Actions.echo(); }}>push clone scene (EchoView)</Button>
+        <Button onPress={() => { Actions.echo(); }}>push clone scene (EchoView)</Button>
```
Comment (mcabs3) [inline Example/components/TabView.js]: What is this for?

Comment (pgonzalez-santiago) [reply Example/components/TabView.js]: Duplicated line, can be removed. My mistake.

Comment (mcabs3) [reply Example/components/TabView.js]: Please, thanks! it will reduce confusion for novice react native devs.

Comment (pgonzalez-santiago) [reply Example/components/TabView.js]: Do you want me to remove the line and push the changes?

Comment (mcabs3) [reply Example/components/TabView.js]: Yes please

Comment (pgonzalez-santiago) [reply Example/components/TabView.js]: Done

**Comment by mcabs3** [comment]
Thanks @pgonzalez-santiago

**Pull request merged**